certs/certs.c

```diff
@@ -4,9 +4,12 @@
 /* This file contains certs needed to communicate with Azure (IoT) */
 
 #include "certs.h"
+//Note: for devices with limited resources, only one cert should be loaded. #defines are used to reduce memory footprint of certificates.
+//For Germany and China regions, please build with -DUSE_GERMANY_CERT_ONLY or -DUSE_CHINA_CERT_ONLY if you wish to load ONLY those certs.
 
 const char certificates[] =
-/* DigiCert Baltimore Root */
+#if !(defined(USE_CHINA_CERT_ONLY) || defined(USE_GERMANY_CERT_ONLY))
```

Comment:
>USE_CHINA_CERT_ONLY [](start = 14, length = 19)

Do we need CMake updates for those defines? Or are you planning to have them in another PR?

Reply:
We do not, if you use none of the defines, the file will contain all certs.